### DISCUSSION TITLE
Assigning surface material

### DISCUSSION THREAD

**Discussion by Somayaji1995** [ansys/pyaedt#4995] [Q&A] [answered]
Hi, I am creating and assigning surface material for my model using pyAEDT. I am using **add_surface_material(name, emissivity=None)** command for creation of new material. Here though i am able to create the material emissivity value of the surface is not being assigned and it is coming as a zero. Can you let me know whether it is a bug or am I making any mistake.
I am using latest version of pyaedt with 2024R2

Script i am using is 
icepak.materials.add_surface_material("Test", emissivity=0.25)

**Comment by Vamsi-Yaddanapudi** [comment]
mat = icepak.materials.add_surface_material("Test_Mat")
mat.emissivity.value = 0.25.

Please try with above solution and it should set the emissivity value properly

**Reply by Samuelopez-ansys** [reply, marked as answer]
I would use:

app = Icepak(version=aedt_version)

mat = app.materials.add_surface_material("Test_Mat2")

mat.emissivity.value = 0.2
